lib/themes/dosomething/paraneue_dosomething/js/components/onboard.js

```diff
@@ -0,0 +1,21 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+
+import Carousel from './carousel.js';
+
+const Onboard = React.createClass({
+  render: function() {
+    return (
+      <div className="onboard">
+        <Carousel />
+      </div>
+    );
+  }
+});
+
+if (localStorage.getItem('dosomething_enable_onboarding') == "true") {
```

Comment:
@katiecrane @sbsmith86 gotcha. good to know. Maybe we can consolidate after? I'm actually using a call to Northstar to grab the user info based on the user id, so that we are pulling the User info from our main User info pool. I'm also adding it as a helper function so we can use that in the API (we repeat a few calls to Northstar there for the info) and so moving forward we just call 1 function and the output could be placed in the Drupal user object in the JS or in the API response via PHP. Thoughts?